modules/jaeger-config-storage.adoc

```diff
@@ -34,109 +35,398 @@ When there is no `es.server-urls` option as part of a Jaeger production instance
 |storage:
   type:
 |Type of storage to use for the deployment.
-|`memory` or `elasticsearch`
+|`memory` or `elasticsearch`.
+Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments as the data does not persist if the pod is shut down. For production environments Jaeger supports Elasticsearch for persistent storage.
 |`memory`
 
-|storage:
-  options:
-   es:
-    server-urls:
+|===
+
+
+.Elasticsearch configuration parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+4+|General Elasticsearch configuration settings
+|elasticsearch:
+  server-urls:
 |URL of the Elasticsearch instance.
-|The fully-qualified domain name of the Elasticsearch server.
+|The fully-qualified domain name of the Elasticsearch server.  If you have specified `storage:type`= `elasticsearch` but have not specified a value for `server-urls` parameter, the Jaeger Operator will use the Elasticsearch Operator to create an Elasticsearch cluster using the configuration in the `storage` section of the custom resource file.
```

Comment:
```suggestion
|The fully-qualified domain name of the Elasticsearch server.  If you have specified `.Spec.Storage.Type`= `elasticsearch` but have not specified a value for `server-urls` parameter, the Jaeger Operator will use the Elasticsearch Operator to create an Elasticsearch cluster using the configuration in the `.Spec.Storage` section of the custom resource file.
```

Reply:
`using the configuration in the `.Spec.Storage.Elasticsearch` section of the custom resource file.